### ISSUE TITLE
graph-node --version lies about the current version

### ISSUE THREAD

**Issue by iameli** [graphprotocol/graph-node#967]
**Do you want to request a *feature* or report a *bug*?**

Bug

**What is the current behavior?**

`graph-node --version` returns `0.1.0` even with version `0.12.0`

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem.**

```
> docker run --rm -it graphprotocol/graph-node:v0.12.0 graph-node --version                                                    11:05:42
graph-node 0.1.0
```

True outside of Docker too:

```
> curl -LO https://github.com/graphprotocol/graph-node/releases/download/v0.12.0/graph-node-linux
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   607    0   607    0     0   1357      0 --:--:-- --:--:-- --:--:--  1357
100 26.5M  100 26.5M    0     0  1327k      0  0:00:20  0:00:20 --:--:-- 2991k
> chmod 755 ./graph-node-linux
> ./graph-node-linux --version
graph-node 0.1.0
```

**What is the expected behavior?**

```
graph-node 0.12.0
```

**Comment by daubaris** [comment]
Makes sense. The version is hardcoded in https://github.com/graphprotocol/graph-node/blob/62e77427e071336750f8b41f2264c9ccae429a82/node/src/main.rs#L127

Because graph-node uses `clap` it could be dynamic by adding `crate_version!` macro.

**Comment by Jannis** [comment]
Closed via #1567.
